awx/__init__.py

```diff
@@ -21,6 +21,30 @@
     MODE = 'production'
 
 
+import django
+import hashlib
+from django.utils.encoding import force_bytes
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.backends.base import schema
+
+class FipsBaseDatabaseSchemaEditor(BaseDatabaseSchemaEditor):
+    
+    @classmethod
+    def _digest(cls, *args):
+        """
+        Generates a 32-bit digest of a set of arguments that can be used to
+        shorten identifying names.
+        """
+        h = hashlib.md5(usedforsecurity=False)
```

Comment:
This parameter is only available on certain platforms carrying this downstream patch (it's not in mainstream Python yet).  We should do something like this instead:

```python
try:
    h = hashlib.md5(usedforsecurity=False)
except TypeError:
    h = hashlib.md5()
```